upi/azure/04_bootstrap.json

```diff
@@ -16,12 +16,6 @@
         "description" : "Bootstrap ignition content for the bootstrap cluster"
       }
     },
-    "sshKeyData" : {
```

Comment:
Unfortunately, I think you are going to need to keep these parameters. The CI jobs will need to keep passing in the parameter so that we can support earlier OpenShift versions that require the parameter. You can provide a default value so that users do not have to provide the parameter. And you should change the description to "unused".

Reply:
Which I know see that @MayXuQQ pointed out the same up above.

Reply:
That makes sense, thanks, i've updated the PR